tests/test_intl.py

```diff
@@ -794,3 +796,101 @@ def test_references(app, status, warning):
     warnings = warning.getvalue().replace(os.sep, '/')
     warning_expr = u'refs.txt:\\d+: ERROR: Unknown target name:'
     yield assert_count(warning_expr, warnings, 0)
+
+
+@with_app(buildername='dummy', testroot='image-glob', confoverrides={'figure_language': 'xx'})
+def test_image_glob_intl(app, status, warning):
```

Comment:
This method is a sequence of the same three instructions over and over again (assert isinstance, assert candidates, assert uri).  Could you please refactor this?